src/components/spinner/Spinner.tsx

```diff
@@ -20,26 +20,34 @@
 
 import { keyframes } from '@emotion/react';
 import styled from '@emotion/styled';
+import classNames from 'classnames';
 import { ReactNode } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { screenReaderOnly } from '~common/helpers/styles';
 
 interface Props {
   className?: string;
+  classNameWrapper?: string;
```

Comment:
using `wrapperClassName` would be less confusing and more accurate IMO

Reply:
Ok I'll change it!